pkg/config/derived-token-file.go

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Package config defines all the configuration parameters. It reads configuration from environment variables and command-line arguments.
```

Comment:
It would be awesome to have this kind of minor changes separated in a different refactor PR so that reviewers can focus on your main work.

Reply:
split to PR: https://github.com/AthenZ/k8s-athenz-sia/pull/187